go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/canonical/microcluster
 
 go 1.22.4
 
+replace github.com/canonical/go-dqlite => github.com/canonical/go-dqlite v0.0.0-20240710170928-54a06bec1336
```

Comment:
Why does this have to be in a `replace` statement? Can't we just update the regular list of dependencies and replace the `github.com/canonical/go-dqlite v1.21.0` line?

Reply:
Seems to get overwritten with `go mod tidy` if we do that.

Reply:
You can run `go get github.com/canonical/go-dqlite@54a06bec1336` to get this exact commit so it doesn't fall back to the latest tag.

Reply:
They have a tagged release with these features now, so problem solved :)